AzerothAuctionAssassin.py

```
@@ -2443,6 +2479,215 @@ def alerts_thread_finished(self):
+        self.pet_ilvl_min_level_input.setText(min_level)
+        self.pet_ilvl_min_quality_input.setText(min_quality)
+        self.pet_ilvl_breeds_input.setText(excluded_breeds.strip("[]"))
+
+        # Update pet name dropdown if possible
+        try:
+            pet_name = self.pet_statistics[
+                self.pet_statistics["itemID"] == int(pet_id)
+            ].iloc[0]["itemName"]
+            index = self.pet_ilvl_name_input.findText(pet_name)
+            self.pet_ilvl_name_input.setCurrentIndex(index)
+        except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)